### PR TITLE
Adds type definitions for to-px

### PR DESCRIPTION
**Pull request by vindhya**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/to-px/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Type definitions for to-px 1.1
+// Project: https://github.com/mikolalysenko/to-px
+// Definitions by: Vindhya Raviraj <https://github.com/vindhya>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function toPX(str: string): number;
+
+declare namespace toPX {
```
Comment (amcasey) [inline types/to-px/index.d.ts:8]: What's the namespace for?  It doesn't seem to be used in the tests or the sample usage in the readme.

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/to-px/index.d.ts (hunk 1 above)
Comment (vindhya) [reply types/to-px/index.d.ts]: Apologies, it's my first time contributing and I thought it was necessary. I've removed the namespace declaration now.

**Review by amcasey — APPROVED** [review]

File: types/to-px/index.d.ts (hunk 1 above)
Comment (amcasey) [reply types/to-px/index.d.ts]: No worries.  Thanks for the update!

**Comment by amcasey** [comment]
The bot seems to have gone rogue, so I'll just merge this manually.

**Pull request merged**